dist/source-map.js

```diff
@@ -1671,6 +1693,10 @@ return /******/ (function(modules) { // webpackBootstrap
 	    throw new Error('Unsupported version: ' + version);
 	  }
 
+	  if (aSourceMapURL) {
+	    sourceRoot = util.join(aSourceMapURL, sourceRoot);
```

Comment:
This is a bit iffy given the source map spec.  Perhaps we should delay this computation.